src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html

```diff
@@ -52,48 +79,66 @@
                     </ul>
 
                     <p>
-                        See <a href="@context.path/documentation/@context.currentDocsVersion/java/JavaPrereqs.html">detailed instructions</a>
-                        for help setting up the prerequisites and getting started. Or, if you are raring to go, just follow these simple steps:
+                        If you have the prerequisites installed and are familiar with Maven, 
+                        create a Hello World project using the commands below. Otherwise, see <a href="@context.path/documentation/@context.currentDocsVersion/java/JavaPrereqs.html">detailed instructions</a>
+                        for help setting up the prerequisites and getting started.
                     </p>
 
                     <ol>
                         <li>
-                            <p>
-                                Create a new project from the command line (all on one line):
-                            </p>
+                          
+                                From a command prompt with administrative privileges, create a new project by entering the following (all on one line):
+                            
 
                             <pre class="prettyprint"><code>mvn archetype:generate -DarchetypeGroupId=com.lightbend.lagom -DarchetypeArtifactId=maven-archetype-lagom-java -DarchetypeVersion=1.3.0</code></pre>
```

Comment:
This command line doesn't match what's in the [Creating and running Hello World with Maven](http://www.lagomframework.com/documentation/1.3.x/java/GettingStartedMaven.html) guide:

> `mvn archetype:generate -Dfilter=com.lightbend.lagom:maven-archetype-lagom-java`

I think the text below matches what is in the getting started guide, rather than what's printed above. Here's the question: do we want this command to be a simple copy & paste that uses default values and doesn't prompt for additional input like the sbt examples on this page and the Scala one?